src/content/events/davos.md

```diff
@@ -13,12 +14,145 @@ location:
+    innovation. Request a meeting at marketing@fil.org to explore opportunities.
+  speakers_list:
+    - name: Marta Belcher
+      title: President & Chair
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/martabelcher/
+      image:
+        src: /assets/images/643e68c10a5fe40dd997f1b0_1-sljgxdvn3ixspjp8v2-i4g.jpeg
+    - name: Megan Klimen
+      title: Founding Officer
+      company: Filecoin Foundation
+      linkedin: "https://www.linkedin.com/in/megan-klimen/"
+      image:
+        src: /assets/images/megan-klimen-davos.webp
+    - name: Clara Tsao
+      title: Founding Officer
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/claratsao/
+      image:
+        src: /assets/images/clara-tsao-davos.webp
+    - name: Danny O'Brien
+      title: Senior Fellow & DWeb Strategy
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/dannyo2/
+      image:
+        src: /assets/images/danny-o-brien-davos.webp
+    - name: Rachel Green Horn
+      title: Head of Marketing and Communications
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/rachelgreenhorn/
+      image:
+        src: /assets/images/rachel-horn-davos.webp
+    - image:
+        src: /assets/images/porter-stowell-davos.jpg
+      name: Porter Stowell
+      title: Head of Ecosystem & Community
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/porterstowell/
+    - name: Aaron Stanley
+      title: Editorial Director
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/aaronstanley27/
+      image:
+        src: /assets/images/aaron-stanley-davos.webp
+    - name: Meredith Pallante
+      title: Head of Events
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/meredith-pallante-cem-cmp-7270549/
+      image:
+        src: /assets/images/meredith-pallante-davos.webp
+    - title: Deputy Head of Marketing and Communications
+      name: Jordan Fahle
+      company: Filecoin Foundation
+      linkedin: https://www.linkedin.com/in/jlfahle/
+      image:
+        src: /assets/images/jordan-fahle-davos.webp
+sponsors:
+  first-tier:
+    - name: GBBC
+      website: https://www.gbbc.io/
+      image:
+        src: /assets/images/gbbc-logo-davos.webp
+    - name: Hedera
+      website: https://hedera.com/
+      image:
+        src: /assets/images/hedera-white-logo-davos.webp
 seo:
   twitter:
     card: summary
-  description: Join FIL.org in Davos for high-level connections and conversations
-    on Web3, decentralization, and the future of the internet. Open daily
-    9am-5pm. Don’t miss this premier event!
-  title: Explore Web3 at Davos | FIL.org
+  description: Join the Filecoin community in Davos during the World Economic
+    Forum (Jan 20-23, 2025). Explore speaking engagements and events shaping the
+    future of the decentralized web.
+  title: FF at Davos 2025 | fil.org
```

Comment:
@mfbgsz The ` | fil.org` part is unnecessary, we automatically include ` | Filecoin Foundation` at the end of the SEO title.

```suggestion
  title: FF at Davos 2025
```

Reply:
sounds good! commiting